tests/orchestrator/test_complete.py

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
IMO e2e tests need to be separated from unit/functional tests, and sit under, say, "tests/e2e" or something similar.